server/controllers/v1/question.js

```diff
@@ -74,9 +74,9 @@ class QuestionController {
     const validator = new Validator();
```

Comment:
Avoid doing validation calls inside the controller. Add the validation method to the route for this controller, to be called before it gets here

Reply:
I created a pivotal tracker story for that here https://www.pivotaltracker.com/story/show/163114836
and I am currently working on it

Reply:
Alright. You can go ahead and merge this